client/components/platform-checkout/save-user/checkout-page-save-user.js

```diff
@@ -61,6 +61,10 @@ const CheckoutPageSaveUser = ( { isBlocksCheckout } ) => {
 		// Take out any non-digit characters, except +.
 		phoneFieldValue = phoneFieldValue.replace( /[^\d+]*/g, '' );
 
+		if ( ! phoneFieldValue.startsWith( '+' ) ) {
+			phoneFieldValue = '+1' + phoneFieldValue;
```

Comment:
Can this truly be assumed? Is the prefix missing only for US phone numbers, or would it be missing for any phone number?

Reply:
The phone input on WooPay already converts numbers without country code to US by default [here](https://github.com/Automattic/woocommerce-payments/blob/8b3acaffac4892988726813ca36938e37a40c887/client/settings/phone-input/index.tsx#L75).